newuser.html

```diff
@@ -0,0 +1,82 @@
+<html>
+    <head>
+
+    </head>
+    <body>
+        <form>
+            <label for="firstName">First Name:</label><br>
+            <input type="text" id="firstName" name="firstName" required minlength="1" maxlength="32">
+            </br>
+            <label for="lastName">Last Name:</label><br>
+            <input type="text" id="lastName" name="lastName" required minlength="1" maxlength="32"><br>
+    
+            <label for="password">Password:</label>
+            </br>
+            <input type="password" id="password" name="password" required>
+            </br>
+            
+            <label for="email">Email:</label>
+            </br>
+            <input type="email" id="email" name="email" required>
+            </br>
+    
+             <label for="telephoneNumber">Telephone Number:</label>
+            </br>
+            <input type="tel" id="telephoneNumber" name="telephoneNumber">
+            </br>
+    
+            <label for="homePage">URL to Home Page:</label>
+            </br>
+            <input type="url" id="homePage" name="homePage">
+            </br>
+    
+            <label for="birthMonth">Birth Month:</label>
+            </br>
+            <input type="month" id="birthMonth" name="birthMonth">
+            </br>
+            
+            <label for="birthYear">Birth Year:</label>
+            </br>
+            <input type="number" id="birthYear" name="birthYear">
+            </br>
+    
+            <label for="emailCheckbox">Can we email you?</label>
+            </br>
+             <input type="checkbox" id="emailCheckbox" name="emailCheckbox">
+            </br>
+    
+
+            <label for="textRadio">Can we text you?</label>
+            <input type = "radio" id = "radioCheckbox1" name = "radioCheckbox">
```

Comment:
For radio buttons, the name attribute for each option should be the same otherwise the user will be able to select multiple options, which defeats the purpose of radio buttons.  So in this case, name=“radioCheckbox” should be the attribute on both options. 

This comment also applies to the radio buttons below.